check-file-existence/action.yaml

```diff
@@ -0,0 +1,56 @@
+name: check-file-existence
+description: ""
+
+inputs:
+  files:
+    description: ""
+    required: true
+  condition:
+    description: ""
+    required: true
+outputs:
+  exists:
+    description: ""
+    value: ${{ steps.check-file-existence.outputs.exists }}
+
+runs:
+  using: composite
+  steps:
+    - name: File check
+      id: check-file-existence
+      run: |
+        set -e
+
+        # Split files into an array
+        tmp="${{ inputs.files }}"
+        input_files=(${tmp})
+
+        # Iterate files
+        found_files=()
+        for f in "${input_files[@]}"; do
+          if [[ -f $f ]]; then
```

Comment:
I think `shellcheck` will fix this.

```suggestion
          if [[ -f "$f" ]]; then
```

Reply:
shellcheck passed this line :thinking: 
Anyway, I will accept this suggestion.

![image](https://user-images.githubusercontent.com/19993104/151964338-4b0fd3ff-7e8c-4be6-909f-8e7e480c92ea.png)

Reply:
Hmm, I should get more familiar with `shellcheck`... :thinking: